test/testdata/packager/lsp_features/bart/bart.rb

```diff
@@ -2,12 +2,12 @@
 
     CatchPhrase = "Don't have a cow, man."
 #   ^^^^^^^^^^^ def: catchphrase
-#   ^^^^^^^^^^^ symbol-search: "Catchphrase", name = "Bart::CatchPhrase", container = "Bart"
+#   ^^^^^^^^^^^ symbol-search: "Catchphrase", name = "CatchPhrase", container = "Bart"
```

Comment:
Perhaps this is a better example; this is `CatchPhrase` defined within the `Bart` package. Searching for `Bart::CatchPhrase` won't work.